cfme/tests/storage/test_object_store_container.py

```diff
@@ -50,3 +51,29 @@ def test_add_remove_tag(container):
     container.remove_tag(added_tag)
     tag_available = container.get_tags()
     assert not tag_available
+
+
+@pytest.mark.manual
+@test_requirements.storage
+def test_storage_object_store_container_edit_tag_openstack():
+    """
+    Polarion:
+        assignee: anikifor
+        caseimportance: medium
+        casecomponent: Cloud
+        initialEstimate: 1/8h
+        startsin: 5.7
+
+        Requires:
+            OpenstackProvider
+
+        Steps:
```

Comment:
This needs to be `testSteps`, and please put `expectedResults` as well, splitting what's here between the two.

Reply:
Apply to all below. Please see `conf/polarion_tools.yaml` for the keys, or other tests for examples.

Reply:
Added for manual tests.
But now I don't actually know what to do when there is multiple steps test and after which test which result is expected.
For automated tests I didn't add expectedResults as I don't think it's needed when it's all documented in the code? @mshriver

Reply:
We don't require testSteps/expectedResults for automation - Just inline comments, and comments as to the content of the test in the regular docblock, out of the Polarion section.

I don't quite follow your other concern, but I think you might be getting at not having expected results for some teststeps.  You can do the following:

```
testSteps:
  1. step one
  2. step two
  3. step three
expectedResults:
  1. thing happens
  2. 
  3. other thing happens
```

Reply:
So expectedResults number should be same number as the step where do I expect that result? So I can have for example 10 steps, but I want expectedResult only for step 7 and 10 so in expected results I will have
```
expectedResults:
    7. something is equal something
    10. something is deleted
```
Like this?

Reply:
95% correct - you still need to have empty `1.`, `2.`, etc

Reply:
@mshriver Reworked it I hope correctly.

Reply:
Looking much better, thanks!! One last comment about empty lines in the docblocks and you'll be good to go.